### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by lautarodragan**
See diff

### CODE REVIEW

**Comment by kennylavender** [comment]
So this info goes in the `squash and merge` comment?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,6 +4,22 @@
...
+* **feat**: A new feature
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing or correcting existing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
+  generation
```
Comment (geoffturk) [inline .github/PULL_REQUEST_TEMPLATE.md]: Does it make sense to add an example or two here so it is clear what the commit message is supposed to look like?

Comment (warrenv) [inline .github/PULL_REQUEST_TEMPLATE.md]: What category would package updates via greenkeeper fall under?

Comment (wzalazar) [reply .github/PULL_REQUEST_TEMPLATE.md]: @geoffturk here all documentation complete about commit messages.

https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits


by default, semantic-release uses the rules of `Angularjs`

Comment (lautarodragan) [reply .github/PULL_REQUEST_TEMPLATE.md]: @geoffturk I think so! @wzalazar we've been using simply `feat: some description`. Is this correct? I gather the Angular way has a `scope` which is used like `feat(scope): some description`, but it looks like scope doesn't make sense for us, right?

Comment (lautarodragan) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good question. I'm guessing it depends on the case — maybe `fix` if the dependency had a bug that could affect poet-js, maybe `chore` if we don't want a new release at all for it (which I think will be most of the cases).

**Comment by lautarodragan** [comment]
@kennylavender that's correct. Regarding breaking changes, I'm wondering which of these, if any, reflects a major version change.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (wzalazar) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think the `scope` is used for monorepos. I'm not sure.

Comment (lautarodragan) [reply .github/PULL_REQUEST_TEMPLATE.md]: Alright. Let's ignore it for now.

**Comment by kennylavender** [comment]
@lautarodragan it looks like you just need to specify breaking changes in the footer of the commit message. It doesn't involve selecting a different type or scope.

https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#footer

This doc has examples: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#footer

**Comment by kennylavender** [comment]
@lautarodragan @warrenv @geoffturk 

I am going to bring up something here I've been want to bring up for a bit. If you guys think its worth bringing up to the rest of the team maybe we can create a discussion or something.

I think the content of the current pull request template needs to be removed. To be honest, when I brought up the idea of a reviewer pr checklist I did not mean for it to be placed in the PR description. I was thinking of just a global public markdown file in some repo that contained a reviewer pr checklist among other things, I was thinking more of a reference for when I am reviewing something but have a feeling I a forgetting something.

Here are a few key reasons why I think the PR checklist should be removed from the pull request template.

- It adds noise, people learn to avoid noise, I have noticed myself finding it annoying and my eyes skip over it.
- It has items that don't apply to all PR's, so I think adds to the reason why my brain sees it as noise.
- It's not being used I am pretty sure, I don't think I have seen anyone use it since the first week or two.
- It doesn't really belong in the PR description, it's more of something a reviewer should copy and paste into a review comment if anything.
- We are now about to add more content to the template, I think its getting a bit out of hand if we add more.

All that being said, I also don't think we should be adding these details provided here to the pull request template either. Those details should be in a global public markdown file for reference, but not here in the pull request template because it adds too much noise. Eventually, as we learn the types and scopes we use regularly we won't need to look them up, but the reference will be there for when we do. We could add a link to the reference in the template if that helps.

I think if we want to use something like semantic release and follow angular commit messages our template should simply make that as simple as possible for the PR creator / merger. Here might be a template we can use for that, this is just a first attempt, I have not created a pr template before so I am unsure if there are any github hooks we can use or anything.

I used single words and underscores instead of spaces so that any word can be double clicked and replaced for quickness.

```
TYPE(SCOPE): SUBJECT

DESCRIPTION_OF_CHANGE

Closes #XXX

BREAKING CHANGE:

DESCRIPTION_OF_BREAKING_CHANGES_DELETE_WITH_KEYWORD_OTHERWISE
```

**Comment by wzalazar** [comment]
@kennylavender 

with your comment the above, maybe if we add a link to a document only?

**Comment by kennylavender** [comment]
@wzalazar yup, I think a link or 2 would be acceptable and a much better solution

**Comment by lautarodragan** [comment]
@kennylavender mostly agreed. Let's move that discussion somewhere else.

Following that, I'll reduce the amount of text in this PR to keep noise-to-signal ratio reasonable. As you guys said, a couple of links should do it.

**Pull request merged**